### ISSUE TITLE
Packaging 

### ISSUE THREAD

**Issue by Manubastro** [rkaczorek/astroberry-server#137]

**Comment by Manubastro** [comment]
Hi,

I'm testing your distribution. I prefer to do an complete install from repository. "https://www.astroberry.io/repo".
It works!

But, Kstars is not starting. I've some mistakes: see capture.
/usr/share/kstarts/.... owner is root 
but in $HOME-astroberry, there isn't .local/share/kstarts
After creating it, it finally works !

![Capture](https://user-images.githubusercontent.com/83101179/115942125-f2ce4f00-a4a8-11eb-878b-af821e54a226.PNG)

**Comment by rkaczorek** [comment]
I don't think it is related to Kstars or Astroberry. I think it is access rights problem.
